auth/src/main/java/com/firebase/ui/auth/ui/phone/SubmitConfirmationCodeFragment.java

```diff
@@ -73,8 +72,10 @@ public static SubmitConfirmationCodeFragment newInstance(FlowParameters flowPara
 
     @Nullable
     @Override
-    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container, @Nullable
-            Bundle savedInstanceState) {
+    public View onCreateView(@NonNull LayoutInflater inflater,
+                             @Nullable ViewGroup container,
+                             @Nullable
```

Comment:
nit: formatter kinda nuked this one.  Can we put the `Bundle savedInstanceState` back on the line with `@Nullable`?

Reply:
yep